src/pyorderly/outpack/location_packit.py

```diff
@@ -168,23 +22,23 @@ def poll_access_token(
 def packit_authorisation(url: str, token: Optional[str]) -> dict[str, str]:
     # If a non-Github token is provided, we assume it is a native Packit token
     # and use that directly.
-    if token is not None and not re.match("^gh._", token):
-        return {"Authorization": f"Bearer {token}"}
+    if token is not None:
+        if re.match("^gh._", token):
+            msg = "Using a GitHub token to login to Packit isn't supported anymore."
```

Comment:
Might be nice to include the additional advice which is in the R: "Either use a Packit token or omit the token to use interactive authentication.."